src/utils/filterUtils.ts

```diff
@@ -0,0 +1,31 @@
+import { FilterTypes } from '../enums';
+import { Status, Todo } from '../types';
+
+const { Active, Completed } = FilterTypes;
+
+export const getFilteredTodos = (
+  todos: Todo[],
+  status: Status,
```

Comment:
The `status` parameter is typed as `Status`, but the function uses `FilterTypes` for comparison. Ensure that the `status` parameter and its comparisons are consistent with the `FilterTypes` enum to avoid potential mismatches.